docs/wiki/research/PBS/PEPC.md

```diff
@@ -0,0 +1,216 @@
+# Protocol-Enforced Proposer Commitments (PEPC)
```

Comment:
I read through the whole PEPC document and I don't have any specific issues to point out. It's great but again, pretty heavy reading with some points like advantages/features which could be coupled to make it more lean.